sass/components/_post.scss

```diff
@@ -1173,10 +1157,47 @@
         }
     }
 
+    .post-pre-header {
+        display: flex;
+        height: 25px;
+        padding-bottom: 10px;
+
+        .post-pre-header__icons-container {
+            display: flex;
+            justify-content: flex-end;
+            align-items: center;
+            padding-right: 9px; // If the padding of post__img changes, this needs to be adjusted accordingly
+            width: 52px; // If the width of post__img changes, this needs to be adjusted accordingly
```

Comment:
Ideally, the values for `padding-right` and `width` should have been `10px` and '53px', respectively, to match those of `post__img`, but the actual rendered value for `post__img` width is actually `52.xxx`. So, these values were the ones that proved proper alignment as shown in the screenshot in the PR description